CountryPicker/CountryPicker/Country.swift

```diff
@@ -46,8 +35,9 @@ open class Country {
     private var image: UIImage?
 
     // MARK: - Functions
-    public init(countryCode code: String) {
+    public init(countryCode code: String, dialingCode: String? = nil) {
         self.countryCode = code
+        if let dialingCode = dialingCode { self.dialingCode = "+\(dialingCode)"}
         countryName = mapCountryName(self.countryCode)
         imagePath = "CountryPickerController.bundle/\(self.countryCode)"
```

Comment:
@EvanCooper9 Since it's required to pass`countryCode` value, wouldn't it be best to perform internally a search for country dial code then assign that value to `dialingCode` property.